packages/@azure/servicebus/data-plane/examples/javascript/gettingStarted/sendWithInteractiveAuth.js

```diff
@@ -0,0 +1,56 @@
+/*
+  This sample demonstrates how to create a namespace using TokenCredentials.
+
+  Setup: AAD RBAC is enabled only on the new namespaces in these regions
+    - [US East, US East 2, or West Europe] for the preview.
+    Please ensure that your servicebus-namespace is present in one of these regions.
+*/
+
+const { Namespace } = require("@azure/service-bus");
+const { interactiveLogin } = require("ms-rest-azure");
+
+const aadServiceBusAudience = "https://servicebus.azure.net/";
```

Comment:
There is no benefit of having this as const i.e it doesnt get used more than once. So, lets reduce a line of code in order to simplify the sample and use this directly

Reply:
Same for the other file